### PR TITLE
DILPREET  | Test | Template Bugs 

### PR DESCRIPTION
**Pull request by dilpreetsinghofficial**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/invoice/templates/edit-template/filters-container/content-filters/content.filters.component.html [hunk 1]
```diff
@@ -32,7 +32,8 @@ <h2>Header</h2>
                         *ngIf="
                             fieldsAndVisibility.header.displayExchangeRate &&
                             customTemplate?.templateType !== 'gst_template_b' &&
-                            customTemplate?.templateType !== 'gst_template_d'
+                            customTemplate?.templateType !== 'gst_template_d' &&
+                            customTemplate.templateType !== 'thermal_template'
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/invoice/templates/edit-template/filters-container/content-filters/content.filters.component.html]:
```suggestion
                            customTemplate?.templateType !== 'thermal_template'
```

File: apps/web-giddh/src/app/invoice/templates/edit-template/filters-container/content-filters/content.filters.component.html [hunk 2]
```diff
@@ -49,7 +50,8 @@ <h2>Header</h2>
                         *ngIf="
                             fieldsAndVisibility.header.displayPlaceOfSupply &&
                             customTemplate?.templateType !== 'gst_template_b' &&
-                            customTemplate?.templateType !== 'gst_template_d'
+                            customTemplate?.templateType !== 'gst_template_d' &&
+                            customTemplate.templateType !== 'thermal_template'
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/invoice/templates/edit-template/filters-container/content-filters/content.filters.component.html]:
```suggestion
                            customTemplate?.templateType !== 'thermal_template'
```

File: apps/web-giddh/src/app/invoice/templates/edit-template/filters-container/content-filters/content.filters.component.html [hunk 3]
```diff
@@ -66,7 +68,8 @@ <h2>Header</h2>
                         *ngIf="
                             fieldsAndVisibility.header.displayPlaceOfCountry &&
                             customTemplate?.templateType !== 'gst_template_b' &&
-                            customTemplate?.templateType !== 'gst_template_d'
+                            customTemplate?.templateType !== 'gst_template_d' &&
+                            customTemplate.templateType !== 'thermal_template'
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/invoice/templates/edit-template/filters-container/content-filters/content.filters.component.html]:
```suggestion
                            customTemplate?.templateType !== 'thermal_template'
```

File: apps/web-giddh/src/app/invoice/templates/edit-template/filters-container/content-filters/content.filters.component.html [hunk 4]
```diff
@@ -83,7 +86,8 @@ <h2>Header</h2>
                         *ngIf="
                             fieldsAndVisibility.header.displayLutNumber &&
                             customTemplate?.templateType !== 'gst_template_b' &&
-                            customTemplate?.templateType !== 'gst_template_d'
+                            customTemplate?.templateType !== 'gst_template_d' &&
+                            customTemplate.templateType !== 'thermal_template'
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/invoice/templates/edit-template/filters-container/content-filters/content.filters.component.html]:
```suggestion
                            customTemplate?.templateType !== 'thermal_template'
```

File: apps/web-giddh/src/app/invoice/templates/edit-template/filters-container/content-filters/content.filters.component.html [hunk 5]
```diff
@@ -1769,6 +1773,7 @@ <h2>Table</h2>
                         <tr
                             *ngIf="
                                 customTemplate?.templateType === 'gst_template_a' &&
+                                customTemplate.templateType !== 'thermal_template' &&
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/invoice/templates/edit-template/filters-container/content-filters/content.filters.component.html]:
```suggestion
                                customTemplate?.templateType !== 'thermal_template' &&
```

File: apps/web-giddh/src/app/invoice/templates/edit-template/filters-container/content-filters/content.filters.component.html [hunk 6]
```diff
@@ -2290,6 +2295,7 @@ <h2>Footer</h2>
                         class="clearfix mr-t1 form-group"
                         *ngIf="
                             customTemplate?.templateType !== 'thermal_template' &&
+                            customTemplate.templateType !== 'thermal_template' &&
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/invoice/templates/edit-template/filters-container/content-filters/content.filters.component.html]: remove this line no use

**Review by dvCodeWorld — CHANGES_REQUESTED** [review]

**Review by dvCodeWorld — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**